DataImport.Common/Constants/IngestionLogLevel.cs

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace DataImport.Common.Constants
```

Comment:
@jagudelo-gap 
Please move this file to Enums folder.

Reply:
@CSR2017 
Done